akka-cluster/src/main/resources/reference.conf

```diff
@@ -239,9 +239,12 @@ akka {
     # Useful for master-worker scenario where all routees are remote.
     allow-local-routees = on
 
+    # Use members with a particular set of roles, or all members if undefined or empty.
```

Comment:
I think it wouldn't hurt to be explicit that a node needs to have all the roles here.